<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="formArea">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="formAreaContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>382</width>
         <height>492</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>dialog_title</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="titleEdit">
          <property name="toolTip">
           <string>dialog_title_edit_help</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string>dialog_description</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="completeTextLabel">
          <property name="text">
           <string>oncomplete_text</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="startValueLabel">
          <property name="text">
           <string>start_value</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="startValueSpin">
          <property name="toolTip">
           <string>start_value_edit_help</string>
          </property>
          <property name="minimum">
           <number>-2147483647</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="endValueLabel">
          <property name="text">
           <string>end_value</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="endValueSpin">
          <property name="toolTip">
           <string>end_value_edit_help</string>
          </property>
          <property name="minimum">
           <number>-2147483647</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="tickIntervalLabel">
          <property name="text">
           <string>tick_interval</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QSpinBox" name="tickIntervalSpin">
          <property name="toolTip">
           <string>tick_interval_edit_help</string>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>600000</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="advanceAmountLabel">
          <property name="text">
           <string>advance_amount</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QSpinBox" name="advanceAmountSpin">
          <property name="toolTip">
           <string>advence_amount_edit_help</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTextEdit" name="descriptionEdit">
          <property name="toolTip">
           <string>dialog_description_edit_help</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTextEdit" name="completeTextEdit">
          <property name="toolTip">
           <string>complete_text_edit_help</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="unitLabel">
          <property name="text">
           <string>value_unit</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="unitEdit">
          <property name="toolTip">
           <string>value_unit_edit_help</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="execButton">
         <property name="text">
          <string>&amp;Execute</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Run">
    <property name="title">
     <string>&amp;Run</string>
    </property>
    <addaction name="action_Execute"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Run"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Execute">
   <property name="text">
    <string>&amp;Execute</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
